core/blockchain/impl/block_tree_impl.cpp

```diff
@@ -1203,13 +1197,9 @@ namespace kagome::blockchain {
 
   outcome::result<void> BlockTreeImpl::reorganize() {
     auto block = BlockTreeImpl::bestLeaf();
```

Comment:
Did https://github.com/soramitsu/kagome/issues/1514 want to change "best" to "finalized"?

Also when "best" goes down, we need to erase mapping above it (e.g. best 10 -> 9).
Cursor `seekLast` and `prev` may help revert/fix that chain (like in "core/network/warp/cache.cpp").

Reply:
> Did #1514 want to change "best" to "finalized"?
Yes, but refactored code makes it unnecessary.

> Also when "best" goes down, we need to erase mapping above it (e.g. best 10 -> 9).
You right, good point. I'll cover it